src/test/java/com/segment/analytics/android/integrations/comscore/ComScoreTest.java

```diff
@@ -593,28 +621,35 @@ public void videoContentPlaying() {
                     .putValue("playbackPosition", 70))
             .build());
 
-    Mockito.verify(streamingAnalytics).startFromPosition(70);
-    Mockito.verify(streamingAnalytics).notifyPlay();
-  }
+
+    Mockito.verify(streamingAnalytics).
```

Comment:
The spacing seems off here. :)

Reply:
Updated spacing in commit https://github.com/segment-integrations/analytics-android-integration-comscore/pull/18/commits/028ab7c132349591686a791e7713a474b87d6bed